src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
-spring.datasource.username=kodilla_user
-spring.datasource.password=kodilla_password
+spring.datasource.url=jdbc:mysql://localhost:3306/ecomercee?serverTimezone=Europe/Warsaw&useSSL=False

```

Comment:
Masz literówkę w nazwie ecommercee powinno być :)

Reply:
To usuwam cały ten plik z pull requesta, bo chyba w Twoim to było wcześniej już zrobione i sobie to ściągnąłem do mastera ;)